editor.planx.uk/package.json

```diff
@@ -12,8 +12,8 @@
     "@mui/lab": "5.0.0-alpha.170",
     "@mui/material": "^5.15.2",
     "@mui/utils": "^5.15.2",
-    "@opensystemslab/map": "^0.8.3",
-    "@opensystemslab/planx-core": "git+https://github.com/theopensystemslab/planx-core#b975cf9",
+    "@opensystemslab/map": "^0.8.2",
```

Comment:
This looks like a mistake when rebasing - we want to maintain the more recent 0.8.3 (matching `main`, and updated here - https://github.com/theopensystemslab/planx-new/pull/3337)

Reply:
Sorted! Cheers